src/Shared/src/ProviderAliasUtilities/Microsoft.Extensions.ProviderAliasUtilities.Sources.csproj

```diff
@@ -0,0 +1,7 @@
+<Project>
```

Comment:
This is fine but only if it is _not_ creating a new (useless) package

Reply:
It is being produced. Will remove.